src/coh3/coh3-types.ts

```diff
@@ -2,6 +2,10 @@ export const raceTypeArray = ["german", "american", "dak", "british"] as const;
 
 export type raceType = (typeof raceTypeArray)[number];
 
+/** The british is different for multiplayer. */
+export const raceMultiplayer = ["german", "american", "british_africa", "afrika_korps"] as const;
+export type raceMultiplayer = (typeof raceMultiplayer)[number];
```

Comment:
Where is this used? Where these show up?:
```
"british_africa", "afrika_korps"
```

Reply:
I see, hmm, can we please rename that variable to `raceAttributesMultiplayer` ? Because `raceMultiplayer` looks to me like it's used in the multiplayer, but the API reports raceType (in some cases 😮‍💨 ) ...

Generally Relic has a mess in all of this shit.

Reply:
All good, will change it ;)